common/utilities/string/wrap.cpp

```diff
@@ -11,25 +11,27 @@
 namespace utilities {
 namespace string {
 
+void trim_trailing_spaces( std::string &remaining_paragraph )
+{
+    auto non_space_index = remaining_paragraph.find_first_not_of( ' ' );
+
+    if( non_space_index != std::string::npos )
+    {
+        // Remove trailing spaces
+        remaining_paragraph = remaining_paragraph.substr( non_space_index );
```

Comment:
This removes the prefix spaces, not the trailing...

Reply:
👍